src/plugins/discover/public/__mocks__/index_pattern.ts

```diff
@@ -6,8 +6,7 @@
  * Side Public License, v 1.
  */
 
-import { IIndexPatternFieldList } from '../../../data/common';
-import { DataView } from '../../../data/common';
+import { IIndexPatternFieldList, DataView } from '../../../data_views/common';
```

Comment:
It would be better to import from `public` when possible. Note: Don't address this quite yet as I'd like to get feedback https://github.com/elastic/kibana/issues/118826#issuecomment-1055630122

Reply:
any update here? seems there's no answer for that, thx!

Reply:
I've pinged the team in slack. Hopefully we will get a response shortly.

Reply:
@kertal Please import from public/server as appropriate.